app/uk/gov/hmrc/agentsubscriptionfrontend/controllers/CheckAgencyController.scala

```diff
@@ -108,11 +111,7 @@ class CheckAgencyController @Inject()
             registrationName = knownFactsResult.taxpayerName,
```

Comment:
Shouldn't we use `AuthorisedAgentWithEmptyEnrolment` here? we do not use `enrolment` anymore.